locales/pl.json

```diff
@@ -1,6 +1,6 @@
 {
-  "HEADER": "Podaj swój email pod spodem aby dołączyć do %s na Slack!",
+  "HEADER": "Podaj swój adres email, aby dołączyć do %s na Slacku!",
   "ENTER_EMAIL": "Podaj swój adres email",
   "ENTER_TOKEN": "Podaj swój kod zaproszenia",
-  "TITLE": "Dołącz do społeczności %s na Slack!"
+  "TITLE": "Dołącz do społeczności %s na Slacku!"
```

Comment:
```suggestion
  "TITLE": "Dołącz do społeczności %s na Slack'u!"
```

Reply:
Disagree.

You add apostrophe, when the word is ending with vowel, eg. iPhone'a, Google'a (and this kind of examples are present in the link you've provided). In other situations you skip it, eg. Facebooka, Slacka, Androida. Writing everyting with the apostrophe would look... weird. Have a look:

"Włączyłem telefon z Android'em i uruchomiłem Slack'a. Na MacBook'u wyłączyłem Twitter'a."

Reply:
Yeah, you're right. Leave it as it is.